src/test/regress/sql/gp_parallel.sql

```diff
@@ -39,6 +39,19 @@ set search_path to test_parallel;
 -- set this to default in case regress change it by gpstop.
 set gp_appendonly_insert_files = 4;
 
+-- CBDB(#131): test parallel_workers during create AO/AOCO table take effect
+begin;
+set local enable_parallel = on;
+create table ao1(x int, y int) using ao_row with(parallel_workers=2);
+create table ao2(x int, y int) using ao_row with(parallel_workers=2);
+create table aoco1(x int, y int) using ao_column with(parallel_workers=2);
+create table aoco2(x int, y int) using ao_column with(parallel_workers=2);
```

Comment:
Add a test about parallel_workers=0 ?

Reply:
done. noted that default value for `parallel_workers` is -1. so it's actually useful to check whether everything fine under `parallel_workers=0`.